roles/ceph-rgw/tasks/common.yml

```
@@ -33,6 +33,11 @@
     - item is not skipped
     - item.item.copy_key | bool
 
+- name: check if haproxy is deployed
+  stat:
+    path: /lib/systemd/system/haproxy.service
+  register: haproxy_service_status
```

```suggestion
  register: haproxy_service_file
```

Service status typically refers to stopped/started/etc. - we're simply checking if the service file exists here.

---

Done.